BlogTemplate/Models/BlogDataStore.cs

```diff
@@ -20,13 +20,44 @@ public BlogDataStore(IFileSystem fileSystem)
         {
             _fileSystem = fileSystem;
             InitStorageFolder();
+            InitCurrentId();
         }
         public void InitStorageFolder()
         {
             _fileSystem.CreateDirectory(StorageFolder);
             _fileSystem.CreateDirectory(DraftsFolder);
         }
 
+        public void InitCurrentId()

```

Comment:
This method could be unit tested.  Also, it should probably be private (same with `InitStorageFolder()`)